src/components/Meta/index.js

```diff
@@ -5,7 +5,10 @@ import { useEffect } from 'react'
 const Meta = ({ title }) => {
   const applicationTitle = 'Hackney Repairs Hub'
   useEffect(() => {
-    hotjar.initialize(3085570, 6)
+    if (!window.Cypress) {
```

Comment:
Is there an easy way to also exclude dev and staging?

Reply:
Perhaps. But is there a need to do so?